libs/shared/account/account/src/lib/account.error.ts

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { BaseError } from '@fxa/shared/error';
+
+export class AccountError extends BaseError {
+  constructor(message: string, cause?: Error) {
```

Comment:
I'm noticing a pattern where a lot of the libraries are starting to branch out and create their own type signatures for the error constructor. Is this okay, or is this something we want to standardize?
I had all of my errors [here](https://github.com/mozilla/fxa/blob/295130c0b3504ee795032c471181c29dce5fd3cc/libs/shared/contentful/src/lib/errors.ts#L20) use the parent error type signature, if we want to copy that pattern elsewhere.

Reply:
We should definitely standardize what we expect this to look like.